bundle/metadata/annotations.yaml

```diff
@@ -15,4 +15,4 @@ annotations:
   operators.operatorframework.io.test.config.v1: tests/scorecard/
 
   # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: v4.15-v4.16
+  com.redhat.openshift.versions: "v4.15-v4.16"
```

Comment:
Why is this change required for this version of the csm-operator? Why wasnit it required for 1.6.0?